app/controllers/admin/snap_applications_controller.rb

```diff
@@ -17,5 +17,25 @@ class SnapApplicationsController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
+    #
+    def resend_fax
+      application = SnapApplication.find(params[:id])
+      application.exports.faxed.succeeded.update_all(
```

Comment:
so we want to update successful faxes to show as failed? I am kind of confused.

I think it being set to failed so that it can re-run. Is there a way to do something differently to make it less confusing? Maybe use a different export method like `admin_fax` that works even if something has already been faxed?